Tests/MetricsTests/TestMetrics.swift

```diff
@@ -166,7 +166,9 @@ internal class TestTimer: TimerHandler, Equatable {
             case .minutes: return (value / 1_000_000_000) * 60
             case .seconds: return value / 1_000_000_000
             case .milliseconds: return value / 1_000_000
+            case .microseconds: return value / 1000
             case .nanoseconds: return value
+            default: preconditionFailure("unknown display unit: \(displayUnit)")
```

Comment:
Should we maybe store a conversion to another unit for every new unit? We could for example require that each new unit defines its conversion to nanoseconds so we don’t have to crash if we don’t know it?

Reply:
Or perhaps as a `default` return `value` without any conversion?

Reply:
@MrLotU that would lead to wrong data, no?

Reply:
Ah. Of course 🤦‍♂ I think in that case your proposed solution is a lot better. Disregard my previous comment.

Reply:
I agree with @weissi , this struct might as well be a Double (or Int64) that expresses how many are in a nanosecond. And if you expose that value then no one has to care about the addition of well named static units ever again.

Reply:
Double is a good idea. We could also make “seconds” (as the SI unit) the base and require everything to be able to express itself in seconds if nothing better is available?

Reply:
good idea: https://github.com/apple/swift-metrics/pull/64/commits/1c4103e92d52b05e356e66e3b1dd2b51b3caba0b